libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -348,27 +348,31 @@ + (NSString *)loginStateDescriptionFromLoginState:(SFUserAccountLoginState)login
 }
 
 NSString *SFKeyForUserAndScope(SFUserAccount *user, SFUserAccountScope scope) {
+    return  SFKeyForUserIdAndScope(user.credentials.userId,user.credentials.organizationId,user.credentials.communityId,scope);
+}
+
+NSString *SFKeyForUserIdAndScope(NSString *userId,NSString *orgId, NSString *communityId, SFUserAccountScope scope) {
     NSString *key = nil;
     switch (scope) {
         case SFUserAccountScopeGlobal:
             key = kGlobalScopingKey;
             break;
             
         case SFUserAccountScopeOrg:
-            if (user.credentials.organizationId != nil) {
-                key = user.credentials.organizationId;
+            if (userId != nil) {
```

Comment:
I'm not sure I understand why this changed. For org level scope, all we need is `orgId` - it should not require any knowledge of `userId`. In this case, `key` should be `orgId`, not `userId`.